torch/utils/data/datapipes/iter/loadfilesfromdisk.py

```diff
@@ -1,28 +1,42 @@
+from io import IOBase
+from typing import Iterable, Tuple
+
 from torch.utils.data import IterDataPipe
 from torch.utils.data.datapipes.utils.common import get_file_binaries_from_pathnames
-from typing import Iterable, Iterator, Tuple
-from io import BufferedIOBase
 
-class LoadFilesFromDiskIterDataPipe(IterDataPipe[Tuple[str, BufferedIOBase]]):
+
+class LoadFilesFromDiskIterDataPipe(IterDataPipe[Tuple[str, IOBase]]):
     r""" :class:`LoadFilesFromDiskIterDataPipe`.
 
-    Iterable Datapipe to load file binary streams from given pathnames,
-    yield pathname and binary stream in a tuple.
+    Iterable Datapipe to load file streams from given pathnames,
+    yield pathname and file stream in a tuple.
     args:
         datapipe: Iterable datapipe that provides pathnames
+        mode: An optional string that specifies the mode in which
+            the file is opened. Default 't' refers to text stream
+            and 'b' refers to binary stream.
         length: a nominal length of the datapipe
     """
 
     def __init__(
             self,
             datapipe : Iterable[str],
+            mode: str = 't',
```

Comment:
Why not keep `"b"` as the default? I imagine that is the most common use case. Otherwise I would go with `"r"` the default of `open()`.

Reply:
The reason I set `t` as default is to mimic the default behavior of `open()`. And since `LoadFileFromDisk` should only be used for reading, I didn't put `r` as the default.
Anyway, I agree with you that binary mode should be the most common use case. And, since I would explicitly write comments about the default argument of `mode`. I would also go with `b`.